web/src/containers/admin/MyAccount.js

```diff
@@ -0,0 +1,165 @@
+import { Alert, Button, Spinner, TextField } from '@cmsgov/design-system-core';
+import PropTypes from 'prop-types';
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { editSelf } from '../../actions/admin';
+import Password from '../../components/PasswordWithMeter';
+import { t } from '../../i18n';
+
+class MyAccount extends Component {
+  state = {
+    success: false,
+    user: {
+      name: '',
+      password: '',
+      phone: '',
+      position: ''
+    }
+  };
+
+  constructor(props) {
+    super(props);
+
+    const {
+      user: { name, phone, position }
+    } = props;
+
+    this.state.user = { name, phone, position };
+  }
+
+  static getDerivedStateFromProps(newProps, prevState) {
+    if (newProps.fetching) {
+      return { success: 0 };
+    }
+    if (prevState.success === 0) {
+      return { success: !newProps.error };
+    }
+    return null;
+  }
+
+  handleEditAccount = e => {
+    const { name, value } = e.target;
+    this.setState(prev => ({ user: { ...prev.user, [name]: value } }));
+  };
+
+  editAccount = e => {
+    e.preventDefault();
+    const { editAccount } = this.props;
+    const { user } = this.state;
+    editAccount(user);
+  };
+
+  goBack = () => {
+    const {
+      history: { goBack }
+    } = this.props;
+    goBack();
+  };
+
+  render() {
+    const { fetching } = this.props;
+    const {
+      success,
+      user: { name, password, phone, position }
+    } = this.state;
+
+    return (
+      <Fragment>
+        <header className="clearfix mb2 px2 py1 bg-white">
+          <div className="left">
+            <Link to="/" className="btn px0 bold caps">
+              {t('titleBasic')}
+            </Link>
+          </div>
+        </header>
+
+        <div className="ds-l-col--5 ds-u-fill--white ds-u-margin-x--auto ds-u-margin-y--3 ds-u-padding--6 ds-u-radius">
+          <div className="ds-u-margin-x--auto" style={{ maxWidth: '460px' }}>
+            {!!success && <Alert variation="success">Changes saved</Alert>}
+
+            <h1 className="ds-h1">Manage account</h1>
+
+            <form onSubmit={this.editAccount}>
+              <TextField
```

Comment:
It looks like `TextField` supports an `ariaLabel` property that we could use to write something more descriptive for screen readers (I'm thinking, "Enter your full name" or "enter your user name," depending on what the name is supposed to be). We could do this with all of them, so it reads something like:
"Enter your full name.
"Please provide a phone number.
"Enter your position [within CMS?]"

Reply:
Nice!  Is this something we should ping James on?  (I'm also not sure what "position" means, it was just in the data model so I made a field for it.  Maybe we don't need it?  @jeromeleecms @NAretakis?)

Reply:
@mgwalker In some of our earlier discussions, I thought the position field was meant to be pulled into this...

![image](https://user-images.githubusercontent.com/34279968/54506410-8a8c0700-48f9-11e9-9926-3fa8ca1ba1cc.png)

If it doesn't get used, we can cut it from layer 1 and bring it back if we find a need for it.

Reply:
Added `ariaLabel` props.  Still not sure what text should be for the position/role input, but I renamed the visual label from "position" to "role".